apps/projects/app/components/Panel/ReviewWork/ReviewWork.js

```diff
@@ -30,7 +30,7 @@ import {
 import workRatings from '../../../utils/work-ratings.js'
 import { DetailHyperText } from '../../../../../../shared/ui'
 
-const ReviewWork = ({ issue, readOnly }) => {
+const ReviewWork = ({ issue, submissionIndex, readOnly }) => {
```

Comment:
`submissionIndex` doesn't match the inputs on  this line: https://github.com/AutarkLabs/open-enterprise/blob/fd1f8f01af801452a8b1b726fd0c3ac5438ba7a8/apps/projects/app/components/Content/IssueDetail/EventsCard.js#L64

Please fix so that the correct work submission is opened in the panel when a link is clicked

Reply:
Fixed!